backends-common/spamassassin/pom.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements. See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.james</groupId>
+        <artifactId>james-backends-common</artifactId>
+        <version>3.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apache-james-backends-spamassassin</artifactId>
```

Comment:
I don't think that's what we put in backends, at least today it just contains storage utilities

Reply:
I don't entirely agree about that.
backend-commons contains utility classes that we need to reuse and that are linked to a given technology.
With this definition, SpamAssassin could very well live on backends-common because it's used in mailets and in mailbox.
But I find weird to mix 3rd party softwares that "integrate" with James (with mailbox extensions and mailets) with technology we build James on.
With time, we'll have a heap of modules in backends-common because the aim of James is to integrate with things and important modules will be lost in the middle.
So my proposal is to have a third-party module, WDYT ? It will be easy to move that module to another git when we need to.

Reply:
I see the point.